spotify.py

```diff
@@ -111,9 +108,16 @@ def generate_final_file():
                "dk", "do", "ec", "ee", "es", "fi", "fr", "gr", "gt", "hk",
                "hn", "hu", "id", "ie", "is", "it", "jp", "lt", "lu", "lv",
                "mc", "mt", "mx", "my", "ni", "nl", "no", "nz", "pa", "pe",
-               "ph", "pl", "pt", "py", "se", "sg", "sk", "sv", "tr", "tw", "uy"]
+               "ph", "pl", "pt", "py", "se", "sg", "sk", "sv", "tr", "tw",
+               "uy"]
 
+    threads = []
```

Comment:
If you are willing to have some parallelism here, please use `concurrent.futures` and an executor instead.